airbyte-integrations/connectors/source-trustpilot/pyproject.toml

```diff
@@ -0,0 +1,27 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "0.1.0"
+name = "source-trustpilot"
+description = "Source implementation for trustpilot."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/trustpilot"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+packages = [ { include = "source_trustpilot" }, {include = "main.py" } ]
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
+
+[tool.poetry.scripts]
+source-trustpilot = "source_trustpilot.run:run"
+
+[tool.poetry.group.dev.dependencies]
+requests-mock = "^1.9.3"
+pytest-mock = "^3.6.1"
+pytest = "^6.1"
```

Comment:
Any reason we stick with pytest 6 and not using pytest 8? I see this in a lot of connectors - is there a known compatibility problem? /cc @girarda

Reply:
The generator as default provides pytest ^6.1, It is quite stable for most of connectors

Reply:
@natikgadzhi we could try upgrading to 8 (outside of the scope for this PR). I've never looked into whether there would be compatibility issues